examples/nlp/language_modeling/conf/megatron_bart_config.yaml

```diff
@@ -0,0 +1,133 @@
+
+model:
+  # model parallelism 
+  micro_batch_size: 4
+  tensor_model_parallel_size: 1
+  pipeline_model_parallel_size: 1 # T5 PP is not supported yet. Use 1 for now.
+
+  # model architecture
+  make_vocab_size_divisible_by: 128 # Pad the vocab size to be divisible by this value for computation efficiency.
+  pre_process: True # add embedding
+  post_process: True # add pooler
+
+  megatron_amp_O2: False # use AMP with O2 style mixed precision instead of native amp on-the-fly weight autocasting.
+
+  seq_length: 512
+  max_position_embeddings: ${.seq_length}
+  num_layers: 12
+  hidden_size: 768
+  ffn_hidden_size: 3072 # Transformer FFN hidden size. Usually 4 * hidden_size.
+  num_attention_heads: 12
+  init_method_std: 0.02 # Standard deviation of the zero mean normal distribution used for weight initialization.')
+  hidden_dropout: 0.1 # Dropout probability for hidden state transformer.
+  attention_dropout: 0.1 # Dropout probability in the attention layer.
+  kv_channels: null # Projection weights dimension in multi-head attention. Set to hidden_size // num_attention_heads if null
+  apply_query_key_layer_scaling: True # scale Q * K^T by 1 / layer-number.
+  layernorm_epsilon: 1e-5
+  persist_layer_norm: True # Use of persistent fused layer norm kernel.
+  gradient_as_bucket_view: True # Allocate gradients in a contiguous bucket to save memory (less fragmentation and buffer memory)
+  encoder_arch: 'transformer'
+  decoder_arch: 'transformer'
+  activation: 'gelu'
+
+  tokenizer:
+    library: 'megatron'
+    type: 'BertWordPieceCase'
+    model: null
+    vocab_file: null
+    merge_file: null
+    num_sentinel_tokens: 100
+
+  # precision
+  native_amp_init_scale: 4294967296 # 2 ** 32
+  native_amp_growth_interval: 1000
+  fp32_residual_connection: False # Move residual connections to fp32
+  fp16_lm_cross_entropy: False # Move the cross entropy unreduced loss calculation for lm head to fp16
+
+  # miscellaneous
+  seed: 1234
+  use_cpu_initialization: False # Init weights on the CPU (slow for large models)
+  onnx_safe: False # Use work-arounds for known problems with Torch ONNX exporter.
+
+  # not implemented in NeMo yet
+  activations_checkpoint_method: null # 'uniform', 'block'
+  activations_checkpoint_num_layers: 1 
+
+  data:
+    # Path to data must be specified by the user.
+    # can override from the CLI: "model.data.data_prefix=[.5,/raid/data/pile/my-bart_00_text_document,.5,/raid/data/pile/my-bart_01_text_document]",
+    # Or see example below:
+    # data_prefix: 
+    #   - .5
+    #   - /raid/data/pile/my-bart_00_text_document
+    #   - .5
+    #   - /raid/data/pile/my-bart_01_text_document
+    data_prefix: ???
+    data_impl: mmap
+    splits_string: 949,45,5
+    seq_length: ${model.seq_length}
+    skip_warmup: True
+    num_workers: 0
+    dataloader_type: single # cyclic
+    masked_lm_prob: 0.15
+    dataset_type: 'bart'
```

Comment:
@michalivne I think there is a bug because `bart` is not listed (no longer listed?) as an acceptable datatype in the T5 model that BART subclasses.

Reply:
Thanks! Fixed with additional few bugs.